src/stacks-hierarchy/navigation-types.ts

```diff
@@ -59,6 +59,11 @@ type FareContractDetailsRouteParams = {
   orderId: string;
 };
 
+type TicketInformationScreenParams = {
+  fareProductTypeConfigType: string;
+  preassignedFareProductId: string | undefined;
```

Comment:
I think the id should be required. Doesn't really make sense to navigate to a ticket info screen if one doesn't have an id for the ticket product to show.

Reply:
Yes it does. It has the generic ticket info section, and potentially mikro mobility benefits.